lib/data/user.ts

```
@@ -57,23 +57,21 @@ export const addUser = async (
 export const getUsersWithMailboxAccess = async (mailboxId: number): Promise<UserWithMailboxAccessData[]> => {
```

ESLint

[error] 'mailboxId' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)